src/core/renderers/webgl/managers/FilterManager.js

```diff
@@ -85,16 +99,32 @@ export default class FilterManager extends WebGLManager
 
         // get the current filter state..
         let currentState = filterData.stack[++filterData.index];
+        const renderTargetFrame = filterData.stack[0].destinationFrame;
 
         if (!currentState)
         {
             currentState = filterData.stack[filterData.index] = new FilterState();
         }
 
+        let fullScreen = this.fullScreen;
+
+        if (renderTargetFrame.width === renderer.screen.width
+            && renderTargetFrame.height === renderer.screen.height)
+        {
+            for (let i = 0; i < filters.length; i++)
+            {
+                if (filters[i].fullScreen)
```

Comment:
is this required? If the bounds match, we should always be full screen?

Reply:
That's our current renderTarget size. If a non-fullscreen filter is above us.